### PR TITLE
Add annotation to NooBaa storage class to describe it

### PR DESCRIPTION
**Pull request by bipuladh**
- Consumed by OCP UI to describe the storage class

### CODE REVIEW

**Review by nimrod-becker — APPROVED** [review]

**Comment by nimrod-becker** [comment]
Thanks for PR :)
Seems like the branch is out of date, can you please rebase ?

File: deploy/obc/storage_class.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
+  annotations:
+    description: Provides Object Bucket Claims(OBCs)
```
Comment (guymguym) [inline deploy/obc/storage_class.yaml:5]: Perhaps better to separate the parenthesis from the last word?
```suggestion
    description: Provides Object Bucket Claims (OBCs)
```

**Comment by bipuladh** [comment]
> Thanks for PR :)
> Seems like the branch is out of date, can you please rebase ?

Updated!

**Review by nimrod-becker — APPROVED** [review]

**Review by guymguym — APPROVED** [review]

**Comment by nimrod-becker** [comment]
tests passed before rebase, no collisions, merging to make the cut

**Pull request merged**